CryptoAnalysis/src/main/java/crypto/analysis/CryptoAnalysisDataFlowScope.java

```diff
@@ -27,6 +29,14 @@ public boolean isExcluded(DeclaredMethod method) {
         JimpleDeclaredMethod jimpleMethod = (JimpleDeclaredMethod) method;
         String declaringClassName = jimpleMethod.getDeclaringClass().getName();
 
+        if (declaringClassName.startsWith(JAVA_IDENTIFIER)) {
+            return true;
+        }
+
+        if (declaringClassName.startsWith(BOUNCY_CASTLE_IDENTIFIER)) {
+            return true;
+        }
```

Comment:
what is with other identifier?

Reply:
You are right, hard coding the identifier is probably not a good solution. I generalized it to exclude all non application classes from the dataflow scope, i.e. SPDS will not go into third party library (including the JDK and BouncyCastle) code and analyze it. This is the exact behaviour as before, however, implemented with the SPDS API. I also added the irgnoredSections option which I forgot